public/js/utils.js

```diff
@@ -4,6 +4,7 @@ import { getJSON } from "@nodesecure/vis-network";
 
 // Import static
 import avatarURL from "../img/avatar-default.png";
+import { portStore } from "../../src/http-server/context";
```

Comment:
You'r not on the server here but on front (it will not work).

Reply:
Yeah 🤦!